packages/wdio-sumologic-reporter/src/index.ts

```diff
@@ -10,11 +10,23 @@ const log = logger('@wdio/sumologic-reporter')
 const MAX_LINES = 100
 const DATE_FORMAT = 'yyyy-mm-dd HH:mm:ss,l o'
 
+interface Options {
+    stdout?: boolean,
+    syncInterval?: number,
+    sourceAddress: string
+}
+
 /**
  * Initialize a new sumologic test reporter.
  */
 export default class SumoLogicReporter extends WDIOReporter {
-    constructor(options) {
+    options: Options
+    unsynced: Array<string>
+    isSynchronising: boolean
```

Comment:
This is a `WDIOReporter` property, let's get #5998 merged and update the branch

Reply:
did you mean #5909 ?

Reply:
yes

Reply:
@christian-bromann  The property in WDIOReported is "isSynchronised" and in sumologic reported we are using "isSynchronising". is there a difference in both the property?

Reply:
`isSynchronised` is the public property is to be implemented by reporters to signalise that the reporter has done its job. `isSynchronising` is a flag only used internaly in the sumologic reporter.
```suggestion
    isSynchronising = false
```